include/oneapi/dpl/internal/distributed_ranges_impl/detail/enumerate.hpp

```diff
@@ -57,11 +58,11 @@ class enumerate_adapter_closure
         using W = std::conditional_t<std::weakly_incrementable<S>, S, std::size_t>;
         if constexpr (rng::sized_range<R>)
         {
-            return rng::views::zip(rng::views::iota(W{0}, W{rng::size(r)}), std::forward<R>(r));
+            return oneapi::dpl::experimental::dr::shp::views::zip(rng::views::iota(W{0}, W{rng::size(r)}), std::forward<R>(r));
```

Comment:
This code is already in "oneapi::dpl::experimental::dr" namespace, use just "shp::"... Please use shorter namespaces also in other places where full namespace is not needed.